thirdparty/dropbear/CMakeLists.txt

```diff
@@ -1,22 +1,45 @@
+# This is pretty much platform-specific... Hard-coded paths and all that...
+if(CERVANTES)
+    set(PLATFORM_KOREADER_DIR "/mnt/private/koreader")
+elseif(KINDLE)
+    set(PLATFORM_KOREADER_DIR "/mnt/us/koreader")
+elseif(KOBO)
+    set(PLATFORM_KOREADER_DIR "/mnt/onboard/.adds/koreader")
+elseif(POCKETBOOK)
+    set(PLATFORM_KOREADER_DIR "/mnt/ext1/applications/koreader")
+else()
+    set(PLATFORM_KOREADER_DIR)
+endif()
+
+set(DEBUG_TRACE 1)
+set(DROPBEAR_DEFPORT "2222")
+set(DROPBEAR_SMALL_CODE TRUE)
+set(DROPBEAR_X11FWD TRUE)
+set(INETD_MODE FALSE)
+# NOTE: Consider setting this to 1 to debug our crazy scp/sftp hack...
+set(LOG_COMMANDS 0)
+# Paths.
+set(DEFAULT_PATH "/sbin:/usr/sbin:/bin:/usr/bin")
+set(DROPBEAR_PIDFILE "settings/SSH/dropbear.pid")
+if(PLATFORM_KOREADER_DIR)
+    set(DBSCP_PATH ${PLATFORM_KOREADER_DIR})
+    set(SFTPSERVER_PATH "${PLATFORM_KOREADER_DIR}/sftp-server")
+endif()
+# Keys.
+set(DSS_PRIV_FILENAME "settings/SSH/dropbear_dss_host_key")
```

Comment:
Just commented https://github.com/koreader/koreader/issues/12637#issuecomment-2411651938

Maybe `/mnt/public/.ssh` for cervantes?

Reply:
I used the same value as before, if we change it, it'll break some existing configs.

Reply:
Ok, nvm as long as it can be made passwordless.